config_defaults_inc.php

```diff
@@ -1065,7 +1065,10 @@
 
 /**
  * Browser language mapping for 'auto' language selection.
+ * An ISO 639-1 language codes, the most general one must be set the last.
```

Comment:
What do you mean by *the most general one must be set the last* ?

Reply:
Language code without country code, i.e. "en" is on the right in "en-us, en-gb, en-au, en".

Reply:
OK, so if I understand correctly, the comment's purpose is to explain the array key's structure, is this correct ?

> ISO 639-1 language codes

639-1 is 2-char codes - I think this is incorrect. If I'm not mistaken, these are actually BCP 47 language codes, which also allow three-letter codes from ISO 639-3 and 639-5 (see https://en.wikipedia.org/wiki/IETF_language_tag).

> the most general one must be set the last

This is not always true, see *chinese_traditional*, *portuguese_brazil* for example.

If the above is correct, may I suggest the following alternative description

`This configuration defines the mapping between a comma-delimited list of related BCP 47 language codes, usually ending with the most generic one (i.e. the primary language without extended subtag), and the corresponding MantisBT language code.`

Reply:
You're quite right, clarified.